corehq/apps/api/resources/v0_5.py

```diff
@@ -28,6 +27,16 @@
 MOCK_BULK_USER_ES = None
 
 
+def user_es_call(domain, q, fields, size, start_at):
+    query = (UserES()
+             .domain(domain)
+             .set_query({"query_string": {"query": q}})
+             .fields(fields)
+             .size(size)
+             .start(start_at))
+    return query.run().hits
```

Comment:
I just wrote a small function to translate the `es_wrapper` call to `ESQuery`, as there's already a test mock that uses that API.